rpcs3/Emu/System.cpp

```diff
@@ -245,14 +245,15 @@ void Emulator::Init(bool add_only)
 	const std::string dev_bdvd = g_cfg_vfs.get(g_cfg_vfs.dev_bdvd, emu_dir); // Only used for make_path
 	const std::string dev_hdd0 = g_cfg_vfs.get(g_cfg_vfs.dev_hdd0, emu_dir);
 	const std::string dev_hdd1 = g_cfg_vfs.get(g_cfg_vfs.dev_hdd1, emu_dir);
-	const std::string dev_flsh = g_cfg_vfs.get_dev_flash();
-	const std::string dev_flsh2 = g_cfg_vfs.get_dev_flash2();
-	const std::string dev_flsh3 = g_cfg_vfs.get_dev_flash3();
+	const std::string dev_flash = g_cfg_vfs.get_dev_flash();
+	const std::string dev_flash2 = g_cfg_vfs.get_dev_flash2();
+	const std::string dev_flash3 = g_cfg_vfs.get_dev_flash3();
 
 	vfs::mount("/dev_hdd0", dev_hdd0);
-	vfs::mount("/dev_flash", dev_flsh);
-	vfs::mount("/dev_flash2", dev_flsh2);
-	vfs::mount("/dev_flash3", dev_flsh3);
+	vfs::mount("/dev_hdd1", dev_hdd1);
```

Comment:
```suggestion
	if (g_ps3_process_info.get_cellos_appname() == "vsh.self"sv)
		vfs::mount("/dev_hdd1", dev_hdd1);
```
@elad335 How about this trick?